domain/domain.go

```diff
@@ -1437,6 +1437,7 @@ func (do *Domain) InitDistTaskLoop(ctx context.Context) error {
 	} else {
 		serverID = disttaskutil.GenerateSubtaskExecID(ctx, do.ddl.GetID())
 	}
+	ctx = context.WithValue(ctx, "serverID", serverID)
```

Comment:
A little bit hacky.
Can we add serverID as a member variable of DispatcherManager and Dispatcher？

Reply:
ok,  I will do so.